pkg/appgw/appgw_test.go

```diff
@@ -671,11 +670,10 @@ var _ = Describe("Tests `appgw.ConfigBuilder`", func() {
 						FrontendPort:            resourceRef(frontendPortID),
 						SslCertificate:          resourceRef(sslCert),
 						Protocol:                n.HTTPS,
-						HostName:                &domainName,
```

Comment:
Do we have the case that rule.Host is specified in ingress spec without hostname-extension annotation? in this case, httpListeners.hostName will be "host1", and httpListeners.hostNames will be ["host1"]?

Reply:
If only rule.Host is specified, we will still set the hostNames property in the HttpListener.
`hostNames = [ "host1" ]`

Basically, we will stop using the `hostName` altogether.